app/src/main/java/com/getcode/util/ApiModule.kt

```diff
@@ -28,14 +28,22 @@ import io.grpc.android.AndroidChannelBuilder
 import io.reactivex.rxjava3.core.Scheduler
 import io.reactivex.rxjava3.disposables.CompositeDisposable
 import io.reactivex.rxjava3.schedulers.Schedulers
+import kotlinx.coroutines.Dispatchers
 import org.kin.sdk.base.network.api.agora.OkHttpChannelBuilderForcedTls12
 import java.util.concurrent.TimeUnit
+import javax.inject.Named
 import javax.inject.Singleton
+import kotlin.coroutines.CoroutineContext
 
 @Module
 @InstallIn(SingletonComponent::class)
 object ApiModule {
 
+    @Provides
```

Comment:
we should probably make an AppDispatchers data class for IO, Main, and Default and inject that into VMs. they can then use `dispatchers.IO`